### ISSUE TITLE
Saving plaintext password

### ISSUE THREAD

**Issue by JPisaBrony** [jghibiki/mopey#63]
In the SongService:
djpon3Credentials = {"username":"DJPon3", "password": "121"}
The current api only accepts text password, I think it should accept hashed passwords as well

**Comment by jghibiki** [comment]
Good idea. How about it looks at the payload and if `"hash":"<hashed password>"` exists it won't care if `password` doesn't and it will use the hash instead.

**Comment by JPisaBrony** [comment]
that would probably be the best way, what i was thinking of originally was doing something like this
https://stackoverflow.com/questions/14300696/check-if-string-is-an-md5-hash
and in the code
if isHash(password):
md5 = password
else
md5 = hashlib.md5(password).hexdigest()

**Comment by jghibiki** [comment]
that could be dangerous though if someone uses a password manager that generates md5 hashes. Which is pretty common for password managers.
